lib/compression.js

```diff
@@ -6,25 +6,37 @@ const Accept = require('@hapi/accept');
 const Bounce = require('@hapi/bounce');
 const Hoek = require('@hapi/hoek');
 
-
 const internals = {
-    common: ['gzip, deflate', 'deflate, gzip', 'gzip', 'deflate', 'gzip, deflate, br']
+    common: [
+        'gzip, deflate, br, zstd',
```

Comment:
It makes sense to update the common encoding with `'gzip, deflate, br, zstd'`, as this is used by both Chrome and Firefox. This can go in a separate PR.